app/src/main/java/com/framgia/fsalon/screen/scheduler/SchedulerPresenter.java

```
@@ -25,26 +48,58 @@ public void onStart() {
+                              String weekInput) {
+        final List<SchedulerSection> sections = new ArrayList<>();
+        final SchedulerSection section = new SchedulerSection();
+        final List<Scheduler> schedulers = new ArrayList<>();
+        Disposable disposable = mRepository.getItemBooking(filterChoice, page, perpage, status,
+            startDate, endDate, monthInput, weekInput)
+            .subscribeOn(Schedulers.io())
+            .observeOn(AndroidSchedulers.mainThread())
+            .doOnSubscribe(new Consumer<Disposable>() {
+                @Override
+                public void accept(@NonNull Disposable disposable) throws Exception {
+                    mViewModel.showLoadMore();
+                }
+            }).subscribeWith(new DisposableObserver<ManageBookingResponse>() {
+                @Override
+                public void onNext(@NonNull ManageBookingResponse manageBookingResponse) {
+                    section.setTitle(convertDate(manageBookingResponse.getDateBook()));
+                }
+
+                @Override
+                public void onError(@NonNull Throwable e) {
+                    mViewModel.onSchedulerFail();
+                    mViewModel.hideLoadMore();
+                }
+
+                @Override
+                public void onComplete() {
+                    mViewModel.onSchedulerSuccessful(sections);
+                }
+            });
+        mCompositeDisposable.add(disposable);
+    }
+
+    @Override
+    public void loadMoreData() {
+        mPage++;
+        getSchedulers(FILTER_DAY, mPage, OUT_OF_INDEX, NON_FILTER, NON_FILTER, NON_FILTER,
+            NON_FILTER, NON_FILTER);
+    }
+
+    private String convertDate(int timeStamp) {
+        if (timeStamp < 0) {
+            return null;
+        }
+        Date date = new Date(timeStamp);
+        SimpleDateFormat format = new SimpleDateFormat("dd-MM-yyyy");
```

WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.